src/UI/words.tsx

```diff
@@ -217,6 +200,41 @@ const dict = {
     `Are you absolutely sure you want to change attribute from ${oldValue} to ${newValue}? This operation can corrupt the instance.`,
   "inventory.error.mermaid": "Error rendering Mermaid diagram",
 
+  /**
+   * Instance Composer text
+   */
+
+  "instanceComposer.noData.errorTitle": "No Data",
+  "instanceComposer.noData.errorMessage": (serviceId) =>
+    `There is no data available to display for the given Id:${serviceId}`,
+  "instanceComposer.noServiceModel.errorTitle": "No Service Model",
+  "instanceComposer.noServiceModel.errorMessage": (serviceName) =>
+    `There is no service model available for ${serviceName}`,
+  "instanceComposer.labelButtonTooltip": "Toggle connection labels",
+  "instanceComposer.addInstanceButtonTooltip":
+    "Add new instance to the canvas.",
+  "instanceComposer.orderDescription": "Requested with Instance Composer",
+  "instanceComposer.errorMessage": "missing Instance Model",
```

Comment:
```suggestion
  "instanceComposer.errorMessage": "The instance attribute model is missing",
```
These are UI messages, we better make full sentences.  Also, why is this key generic, and the one for noServiceModel specific? 

I feel like the casing of the UI-strings is also inconsistent.